test/multiverse/suites/grpc/grpc_client_test.rb

```diff
@@ -9,107 +9,184 @@
 class GrpcTest < Minitest::Test
   include MultiverseHelpers
 
-  def setup
+  # Helpers
+  TRACE_WITH_NEWRELIC = :@trace_with_newrelic
+  HOST = '0.0.0.0:5000'
+  CHANNEL = :this_channel_is_insecure
+  METHOD = 'routeguide.RouteGuide/GetFeature'
+
+  def exception_class
+    GRPC::Unknown
```

Comment:
This one uses `GRPC` and `basic_grpc_client` uses `::GRPC`. I think they should both use the same approach.

Reply:
Great catch!